db/schema.rb

```diff
@@ -138,32 +151,8 @@
     t.index ["year"], name: "index_stims_on_year"
   end
 
-  create_table "students", force: :cascade do |t|
-    t.string "first_name", null: false
-    t.string "last_name", null: false
-    t.string "preferred_name"
-    t.integer "grad_year", null: false
-    t.integer "osis", null: false
-    t.string "email", null: false
-    t.string "phone", null: false
-    t.integer "team_id"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.integer "user_id"
-    t.boolean "is_newbie", default: true, null: false
-    t.boolean "is_active", default: true, null: false
-    t.index ["email"], name: "index_students_on_email", unique: true
-    t.index ["first_name"], name: "index_students_on_first_name"
-    t.index ["grad_year"], name: "index_students_on_grad_year"
-    t.index ["is_active"], name: "index_students_on_is_active"
-    t.index ["is_newbie"], name: "index_students_on_is_newbie"
-    t.index ["last_name"], name: "index_students_on_last_name"
-    t.index ["osis"], name: "index_students_on_osis", unique: true
-    t.index ["phone"], name: "index_students_on_phone"
-    t.index ["preferred_name"], name: "index_students_on_preferred_name"
-    t.index ["team_id"], name: "index_students_on_team_id"
-    t.index ["user_id"], name: "index_students_on_user_id"
-  end
+# Could not dump table "students" because of following StandardError
+#   Unknown type 'gender' for column 'string'
```

Comment:
Some sort of problem here. You need to reset the db and re-migrate on this branch without the gender changes